### PR TITLE
feat: handle unhandled promise rejections in tests

### PR DESCRIPTION
**Pull request by jschfflr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -104,7 +104,7 @@
     "minimist": "1.2.0",
     "mocha": "9.1.3",
     "ncp": "2.0.0",
-    "pixelmatch": "4.0.2",
+    "pixelmatch": "5.2.1",
```
Comment (OrKoN) [inline package.json]: why is dep update needed?

**Review by OrKoN — APPROVED** [review]

**Comment by GetBentGaming** [comment]
Hello I'm still trying to fix Puppateers mess ups on my computer

**Comment by GetBentGaming** [comment]
He keeps changing my elements

**Comment by GetBentGaming** [comment]
How !!!!!

**Comment by jschfflr** [comment]
@GetBentGaming Feel free to create an issue and to describe your situation there. Happy to take a look if it's something caused by Puppeteer itself.

**Pull request merged**